util/syspolicy/handler.go

```diff
@@ -19,6 +19,8 @@ type Handler interface {
 	ReadString(key string) (string, error)
 	// ReadUInt64 reads the policy settings uint64 value given the key.
 	ReadUInt64(key string) (uint64, error)
+	// ReadBool reads policy settings that are 0 or 1 and converts them to boolean given the key.
```

Comment:
Windows mapping 0 vs not-0 to boolean is an OS quirk, I don't think we should make that part of the interface.

```suggestion
	// ReadBool reads the policy setting's boolean value, given the key.
```